pkg/kubelet/config/config.go

```diff
@@ -467,3 +472,16 @@ func bestPodIdentString(pod *api.Pod) string {
 	}
 	return fmt.Sprintf("%s.%s", name, namespace)
 }
+
+func copyPods(sourcePods []*api.Pod) []*api.Pod {
```

Comment:
Sure. In fact I learned how to do deep copy from `MergedState()`. I should modify it, too. :)